docs/product-assembly_case-management-getting-started/src/main/asciidoc/case-management-stage-activation-and-completion-conditions-proc.adoc

```diff
@@ -1,2 +1,23 @@
-[id='']
-= TODO
\ No newline at end of file
+[id='case-management-stage-activation-and-completion-conditions-proc']
+= Stage activation and completion conditions
+
+Stages can be triggered by a signal event from an end node or the completion of another task node in the case definition, or it can be activated using property configuration.
```

Comment:
they can be triggered by end node, intermediate node or manually via API call

Reply:
changed.